### PR TITLE
feat: task improvements

### PR DESCRIPTION
**Pull request by Davidmattei**
|Q              |A  |
|---------------|---|
|Bug fix?       |n|
|New feature?   |n|	
|BC breaks?     |n|	
|Deprecations?  |n|	
|Fixed tickets  |n|	

- [x] Change icon for history
- [x] Update button label "Send for validation" -> "Send" 
- [x] Task dashboard table no button group 
- [x] Add task actions in modal 
- [x] Show versionTag column (only if contentType with tasks and version tags enabled)

### CODE REVIEW

**Review by theus77 — APPROVED** [review]

File: src/Resources/views/revision/task/ajax.twig [hunk 1]
```diff
@@ -111,6 +111,29 @@
     </div>
 {% endblock %}
 
+{% block modalFooterActions %}
+    <div class="pull-left">
+        <button type="button" class="btn btn-sm btn-default" data-dismiss="modal">Close</button>
```
Comment (theus77) [inline src/Resources/views/revision/task/ajax.twig:116]: Hardcoded close label

File: src/Resources/views/revision/task/columns.twig [hunk 2]
```diff
@@ -29,29 +30,18 @@
 
 {%- block actions -%}
     <div class="btn-group">
-        <button type="button" class="btn btn-sm btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            action <span class="caret"></span>
```
Comment (theus77) [inline src/Resources/views/revision/task/columns.twig:33]: Hardcoded action label

Comment (Davidmattei) [reply src/Resources/views/revision/task/columns.twig:33]: This was deleted

File: src/Resources/views/revision/task/ajax.twig (hunk 1 above)
Comment (Davidmattei) [reply src/Resources/views/revision/task/ajax.twig]: 77b1fb3

File: src/Resources/views/revision/task/ajax.twig [hunk 3]
```diff
@@ -21,14 +21,14 @@
...
-                <button id="btn-validation-request" class="btn btn-sm btn-primary"> <i class="fa fa-paper-plane"></i>&nbsp;{{ 'task.validation.request'|trans  }}</button>
+                <button id="btn-validation-request" class="btn btn-sm btn-primary"> <i class="fa fa-paper-plane"></i>&nbsp;{{ 'task.validation.send'|trans  }}</button>
             </div>
         {% endif %}
         <div class="clearfix">
             <button class="btn-modal-history-task btn btn-sm btn-default pull-right"
                     data-url="{{ path('ems_core_task_ajax_modal_history', {'revisionId': revisionId, 'taskId': task.id }) }}"
                     data-title="{{ 'task.history'|trans }} : {{ task.title }}">
-                    <i class="fa fa-comments" aria-hidden="true"></i>
+                    <i class="fa fa fa-history" aria-hidden="true"></i>
```
[the remark this answers is not in the document]
Comment (Davidmattei) [reply src/Resources/views/revision/task/ajax.twig]: 77b1fb3
